src/builtin.jq

```diff
@@ -255,6 +255,9 @@ def walk(f):
     | f;
   w;
 
+# compact removes nulls from an array or an object
+def compact: del(..|nulls);
```

Comment:
style seem to be whitespace around operators

Reply:
Addressed in https://github.com/jqlang/jq/pull/2838/commits/58a5d09dbdd2b89a1eeb15468c48b32f2f5f4095